front/lib/api/assistant/configuration/table_query.ts

```diff
@@ -55,6 +57,10 @@ export async function fetchTableQueryActionConfigurations({
               model: Workspace,
               as: "workspace",
             },
+            {
```

Comment:
Seen IRL, but was probably better before when logic was in the UI, since this includes will impact many many fetches simply to fix the Assistant Details situation.